data/discovery/field/membership-number.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
```

Comment:
This field won't be needed if the `citizens-advice-bureau-membership-number` register isn't needed.